apps/bfd-pipeline/bfd-pipeline-rda-bridge/pom.xml

```diff
@@ -63,7 +63,7 @@
         <dependency>
             <groupId>org.freemarker</groupId>
             <artifactId>freemarker</artifactId>
-            <version>2.3.31</version>
+            <version>2.3.32</version>
```

Comment:
IMO we should pull all these misc version numbers into the parent pom so our versions are all together.

Reply:
Fixed